### PR TITLE
Fixing TorpedoStore bugs

### PR DESCRIPTION
**Pull request by RudolfAndrasDaniel**
I've fixed several bugs in TorpedoStore.

As an extra, I've placed and intentional bug for educational purposes which is need to be fixed.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -29,25 +31,26 @@ public TorpedoStore(int numberOfTorpedos){
...
+      this.torpedoCount -= numberOfTorpedos;
       success = true;
     } else {
       // simulated failure
       success = false;
     }
 
+    // Intentional bug for educational purposes
+    success = false;
+
```
Comment (AkosUzonyi) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:53]: This is a mission critical system, please do not insert educational code here :D.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  Random random = new Random();
+
```
Comment (AkosUzonyi) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:18]: This variable can be private.

**Review by AkosUzonyi — CHANGES_REQUESTED** [review]

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (RudolfAndrasDaniel) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: I'm sorry, I'll fix it immediately.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (RudolfAndrasDaniel) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Yes, you're right. I'll fix that. The bug made for educational purposes didn't distract you. :D

Comment (AkosUzonyi) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Nearly missed it though ;). Nice try :D

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (AkosUzonyi) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Don't worry, instead of the torpedos, YOU are fired now :D.

**Review by AkosUzonyi — APPROVED** [review]
This was your last contribution to this project... But appoved now :D

**Pull request merged**